charts/synapse/values.yaml

```diff
@@ -113,7 +113,7 @@ ingress:
   enabled: false
   # ingress.annotations -- Annotations to add to the synapse server ingress
   annotations:
-    kubernetes.io/ingress.class: nginx-ingress
+    kubernetes.io/ingress.class: nginx
```

Comment:
This is actually the default ingress class that I use in all of my clusters, so I typically prefer not to change this default. This is easily worked around by using `--set` when installing, or using an infrastructure as code tool to manage these values.

I would also guess that most people don't use the default value below this for the cert-manager cluster-issuer name.

Reply:
I thought it would make sense touse the default as a default :P 
I will undo this later today then.